### PR TITLE
Review header step

### PR DESCRIPTION
**Pull request by jomaora**
https://trello.com/c/pmEt3ZcY/2503-or-composant-question-steps

**Detailed purpose of the PR**

Adding an atom : `ReviewHeaderStepItem`
And a molecule using this atom `ReviewHeaderSteps`

**Result and observation**

Kapture 2022-03-10 at 18 03 04

OK on IE 11

**Testing Strategy**

- Manual testing

### CODE REVIEW

File: packages/@coorpacademy-components/src/atom/review-header-step-item/index.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        icon === 'right' ? style.right : null,
+        icon === 'wrong' ? style.wrong : null,
+        current ? style.current : style.default
+      )}
+    >
+      {content}
+    </div>
+  );
+};
+
```
Comment (Fausto95) [inline packages/@coorpacademy-components/src/atom/review-header-step-item/index.js:47]:
```suggestion
const Content = ({icon, current, value}) => {
  if (!icon) {
    return <span className={style.value}>{value}</span>;
  }

  if (icon === 'right') {
    return (
      <span className={style.value}>
        <RightIcon className={current ? style.currentRightIcon : style.rightIcon} />
      </span>
    );
  }

  if (icon === 'wrong') {
    return (
      <span className={style.value}>
        <WrongIcon className={current ? style.currentWrongIcon : style.wrongIcon} />
      </span>
    );
  }
};

const ReviewHeaderStepItem = props => {
  const {icon, current, value} = props;
  return (
    <div
      className={classnames(
        icon === 'right' ? style.right : null,
        icon === 'wrong' ? style.wrong : null,
        current ? style.current : style.default
      )}
    >
      <Content icon={icon} current={current} value={value} />
    </div>
  );
};

```

**Review by Fausto95 — APPROVED** [review]
`getContent`should be a React component since it returns jsx

File: packages/@coorpacademy-components/src/atom/review-header-step-item/index.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+const ReviewHeaderStepItem = props => {
+  const {icon, current, value} = props;
+  const content = getContent(icon, current, value);
+  return (
+    <div
+      className={classnames(
+        icon === 'right' ? style.right : null,
+        icon === 'wrong' ? style.wrong : null,
+        current ? style.current : style.default
```
Comment (StefanoA1) [inline packages/@coorpacademy-components/src/atom/review-header-step-item/index.js:40]: est-ce que tu peux ajouter un `data-name`? ou cas ou on a besoin pour les test après

File: packages/@coorpacademy-components/src/molecule/review-header-steps/index.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import map from 'lodash/fp/map';
+import ReviewHeaderStepItem from '../../atom/review-header-step-item';
+import style from './style.css';
+
+const ReviewHeaderStep = props => {
+  const {steps} = props;
+
+  const items = map(step => <ReviewHeaderStepItem {...step} key={step.value} />, steps);
+
+  return <div className={style.wrapper}>{items}</div>;
```
Comment (StefanoA1) [inline packages/@coorpacademy-components/src/molecule/review-header-steps/index.js:12]: pareil

**Review by StefanoA1 — APPROVED** [review]

**Review by Fausto95 — APPROVED** [review]

**Pull request merged**